<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerDialog</class>
 <widget class="QDialog" name="CustomerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>441</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="leCardNumber">
      <property name="placeholderText">
       <string>number of card</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="leFirstname">
      <property name="placeholderText">
       <string>firstname</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lePatronymic">
      <property name="placeholderText">
       <string>patronymic</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leBalance">
      <property name="placeholderText">
       <string>balance</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pbCreate">
      <property name="text">
       <string> Створити</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="leID">
      <property name="placeholderText">
       <string>id</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="leLastname">
      <property name="placeholderText">
       <string>lastname</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leAddress">
      <property name="placeholderText">
       <string>address</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
